lib/global-styles.php

```diff
@@ -535,40 +535,49 @@ function gutenberg_experimental_global_styles_normalize_schema( $tree ) {
 	return $tree;
 }
 
-/**
- * Returns the stylesheet resulting of merging
- * core's, theme's, and user's origins.
- *
- * @return string
- */
-function gutenberg_experimental_global_styles_get_stylesheet() {
-	$gs_merged = array();
-	$gs_core   = gutenberg_experimental_global_styles_get_core();
-	$gs_theme  = gutenberg_experimental_global_styles_get_theme();
-	$gs_user   = gutenberg_experimental_global_styles_get_user();
+function gutenberg_experimental_global_styles_get_merged_origins() {
```

Comment:
Can we add a PHP Doc to explain what this function is doing.

Reply:
:+1: https://github.com/WordPress/gutenberg/pull/24275/commits/9aafb6e0f99ad1968e4740ce56326d4c2f4914cf